docs/setup/configuring-logging.asciidoc

```diff
@@ -21,9 +21,9 @@ __<<logging-layouts,Layouts>>__ define how log messages are formatted and what t
 [[log-level]]
 === Log level
 
-Currently we support the following log levels: _all_, _fatal_, _error_, _warn_, _info_, _debug_, _trace_, _off_.
+Currently we support the following log levels: _off_, _fatal_, _error_, _warn_, _info_, _debug_, _trace_, _all_.
 
-Levels are ordered, so _all_ > _fatal_ > _error_ > _warn_ > _info_ > _debug_ > _trace_ > _off_.
+Levels are ordered, so _off_ > _fatal_ > _error_ > _warn_ > _info_ > _debug_ > _trace_ > _all_.
 
 A log record will be logged by the logger if its level is higher than or equal to the level of its logger. Otherwise, the log record is ignored.
```

Comment:
```suggestion
A log record will be logged by the logger if its level is higher than or equal to the level of its logger. For example:
If the output of an API call is configured to log at the `info` level and the parameters passed to the API call are set to `debug`, with a global logging configuration in `kibana.yml` set to `debug`, both the output _and_ parameters are logged. If the log level is set to `info`, the debug logs are ignored, meaning that you'll only get a record for the API output and _not_ for the parameters.

Logging set at a plugin level is always respected, regardless of the `root` logger level. In other words, if root logger is set to fatal and pluginA logging is set to `debug`, debug logs are only shown for pluginA, with other logs only reporting on `fatal`.
```

Real example (please don't use)

With logging configured as 
``` // kibana.yml
logging:
  root:
      appenders: [file]
      level: warn
  loggers:
    - name: savedobjects-service
      level: debug
      appenders: [so-console]
    - name: http.server.response
      level: info
      appenders: [http-console]
```
and the `so-console` appender's pattern is: `"SAVED_OBJECTS_SERVICE--[%date][%level][%logger]---%message"`

We'll get logs like:
```
SAVED_OBJECTS_SERVICE--[2024-04-17T13:05:19.287-07:00][DEBUG][savedobjects-service]---[.kibana] CHECK_VERSION_INDEX_READY_ACTIONS -> DONE. took: 0ms.
SAVED_OBJECTS_SERVICE--[2024-04-17T13:05:19.287-07:00][INFO ][savedobjects-service]---[.kibana] Migration completed after 43ms```

Reply:
@herrBez I've added a release not lable. You'll need to take care of updating the branch